.github/workflows/aws-CD.yml

```diff
@@ -48,4 +48,4 @@ jobs:
             docker kill $(docker ps -q)
             docker rmi $(docker images -q) -f
             yes | docker system prune
-            yes | docker-compose -f docker-compose.prod.yml up --build -d
+            yes | docker compose -f docker-compose.prod.yml up --build -d
```

Comment:
Was removing this dash intentional?

Reply:
Yeah, I'm using the updated `docker compose` ([stackoverflow](https://stackoverflow.com/questions/66514436/difference-between-docker-compose-and-docker-compose) thread)